_faq/security/auth-mix-and-match.md

```diff
@@ -2,7 +2,26 @@
 title: Can I enable Istio Auth with some services while disable others in the same cluster?
 weight: 30
 ---
-Starting with release 0.3, you can use service-level annotations to disable (or enable) Istio Auth for particular service-port.
+{% include home.html %}
+
+Starting with Istio 0.8, you can use [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) to enable (or disable) mTLS per service. For example, the policy below will disable mTLS on port 9080 for service `details`
+
+```command
+$ cat <<EOF | istioctl create -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "example"
+spec:
+  targets:
+  - name: details
+    ports:
+    - number: 9080
+  peers:
+EOF
+```
+
+For older version of Istio (but newer than 0.3), you can use service-level annotations to disable (or enable) Istio Auth for particular service-port.
```

Comment:
version -> versions
particular -> a particular service and port pair.

Reply:
Done